server/mdm/android/service/service.go

```diff
@@ -61,8 +61,10 @@ func NewService(
 ) (android.Service, error) {
 	var client androidmgmt.Client
 	if os.Getenv("FLEET_DEV_ANDROID_GOOGLE_CLIENT") == "1" || strings.ToUpper(os.Getenv("FLEET_DEV_ANDROID_GOOGLE_CLIENT")) == "ON" {
+		fmt.Println("Using google client", os.Getenv("FLEET_DEV_ANDROID_GOOGLE_CLIENT"))
```

Comment:
@MagnusHJensen Are these print statements kept intentionally?

Reply:
Nope, forgot to remove it.